numpy/core/src/multiarray/array_method.c

```diff
@@ -291,6 +291,9 @@ fill_arraymethod_from_slots(
             case NPY_METH_get_reduction_initial:
                 meth->get_reduction_initial = slot->pfunc;
                 continue;
+            case NPY_METH_contiguous_indexed_loop:
+                meth->contiguous_indexed_loop = slot->pfunc;
+                continue;
```

Comment:
Coverage correctly complains that this line is not used.

Reply:
let's not worry about things that are not reachable in normal API for now.  On this one, I am wondering if we should directly go to a `get_contiguous_indexed_loop` to allow easier expansion.  OTOH, that would mean a bit more set-up work.

In either case, I am fine with this, because I think we can reasonably swap it out even if public (few will use it and a brief backcompat layer is probably easy to achieve).

Reply:
Let's do it later if needed